crates/swc_ecma_lexer/Cargo.toml

```diff
@@ -42,11 +42,9 @@ phf                   = { workspace = true, features = ["macros"] }
 swc_atoms             = { version = "5.0.0", path = "../swc_atoms" }
 swc_common            = { version = "10.0.0", path = "../swc_common" }
 swc_ecma_ast          = { version = "10.0.0", path = "../swc_ecma_ast" }
+swc_ecma_utils        = { version = "14.0.0", path = "../swc_ecma_utils" }
```

Comment:
Doesn't swc_ecma_utils depend on `swc_ecma_parser`?
You should rather move utility functions to `swc_comon` instead. Otherwise cargo will refuse to publish